parlai/tasks/dream/agents.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from parlai.core.teachers import FixedDialogTeacher
+from .build import build
+import os
+import json
+
+
+def setup_data(opt, jsons_path):
```

Comment:
Would be cleaner to have these 3 functions (setup_data, num_examples, get) be methods of `DREAMTeacher`